bindings/java/src/tests.cmake

```diff
@@ -48,6 +48,7 @@ set(JAVA_INTEGRATION_TESTS
   src/integration/com/apple/foundationdb/DirectoryTest.java
   src/integration/com/apple/foundationdb/RangeQueryIntegrationTest.java
   src/integration/com/apple/foundationdb/BasicMultiClientIntegrationTest.java
+  src/integration/com/apple/foundationdb/CycleMultiClientIntegrationTest.java
```

Comment:
In the Go code, consider listing `java/src/integration/com/apple/foundationdb/*Test.*`, and telling it to run each test automatically.  That way, as this suite expands, the tests will automatically run in Kubernetes.

From what I can tell, cmake makes this difficult (I'd love to see a counterexample, so we can stop updating these stanzas), but there's no reason for the non-cmake test runners to be bug-for-bug compatible.

(Don't do this for this PR.)

Reply:
yeah, will do in a separate PR, noted it down.